apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -2,6 +2,15 @@
 
 {{description}}
 
+# Class renaming in [`v1.0.0-beta16`](https://www.nuget.org/packages/Google.Cloud.PubSub.V1/1.0.0-beta16)
+
+The following classes have been renamed when moving from version 1.0.0-beta15 to 1.0.0-beta16:
+
+* `PublisherClient` is now named [`PublisherServiceApiClient`](obj/api/Google.Cloud.PubSub.V1.PublisherServiceApiClient.yml)
+* `SubscriberClient` is now named [`SubscriberServiceApiClient`](obj/api/Google.Cloud.PubSub.V1.SubscriberServiceApiClient.yml)
+* `SimplePublisher` is now named [`PublisherClient`](obj/api/Google.Cloud.PubSub.V1.PublisherClient.yml)
+* `SimpleSubscriber` is now named [`SubscriberClient`](obj/api/Google.Cloud.PubSub.V1.SubscriberClient.yml)
```

Comment:
Possibly mention that the settings classes have changed too? (But I think it's okay to not give details - it's obvious enough.)

Reply:
Good point, added.